extensions/lark/src/components/space-list-item.tsx

```diff
@@ -32,7 +32,7 @@ export const SpaceListItem: React.FC<SpaceListItemProps> = ({ node, owner, actio
       node.type === NodeType.Box
         ? `space-icons/type-${node.type}-${node.subtype}.png`
         : `space-icons/type-${node.type}.png`;
-    accessoryTitle = node.author;
+    accessoryTitle = `${node.author} ${dayjs(node.edit_time * 1000).format('YYYY-MM-DD')}`;
```

Comment:
Why do this? Could you please provide a screenshot?

Reply:
@tmkx  This shows the latest editing time. You can judge whether this document is the latest or old, so you don't need to open the old one.

<img width="741" alt="image" src="https://user-images.githubusercontent.com/1717023/163167412-ef329dc9-6ba7-470c-9471-237f42270808.png">

Reply:
If the author's name is very long (maybe an English name or a minority's), the `accessoryTitle` will be too wide:
![image](https://user-images.githubusercontent.com/13805071/163186472-7884e16f-551d-4072-a70c-f65e7d65b864.png)

We can sort the search result by updated time to solve your doubt. I may make it more convenient by adding an overview panel later.

Reply:
@tmkx  Yes, you are right. Maybe It would be better to have a text box here. 

<img width="757" alt="image" src="https://user-images.githubusercontent.com/1717023/163202170-22d58fc0-8a65-40b5-98e9-8b203c3751c8.png">

Reply:
I think so, could you revert this first? I'll redesign it in few days.